File: abc/abc240/a.java
// https://atcoder.jp/contests/abc240/submissions/41451176

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt() - 1;
    int b = stdIn.nextInt() - 1;
    
    if((a + 1) % 10 == b || (a + 10 - 1) % 10 == b) {
      System.out.println("Yes");
    } else {
      System.out.println("No");
    }
  }
}

File: abc/abc240/a.py
# https://atcoder.jp/contests/abc240/submissions/34672562

x = list(map(int, input().split()))
x.sort()
print("Yes" if (x[0] < 2 and x[1] > 9) or x[1] - x[0] == 1 else "No")
